cirq/ops/pauli_string.py

```diff
@@ -272,6 +284,14 @@ def __bool__(self):
     def __len__(self) -> int:
         return len(self._qubit_pauli_map)
 
+    def _repr_pretty_(self, p: Any, cycle: bool) -> None:
```

Comment:
can you explain? is this because jupyter shows the __repr__ by default which is way too long because of the need for roundtrippability?

I have some prototype code that refines `_repr_latex_` on paulistring which looks pretty nice

Reply:
Yeah, it's because the representation is shorter this way.